app/models/offer.rb

```diff
@@ -9,17 +8,47 @@ class Offer < ApplicationRecord
         foreign_key: :job_id
 
     # note: promotions, interns. Are we counting them?
-    # lets add styling
-=begin
-    RETURN: all full-time offers that were accepted for a given month within a year. 
-    note -- does not count conversions (eg. CX Associate Temp to Full-time)
-    YEAR: integer
-    MONTH: integer
-=end
-    def self.get_accepted_offers(year, month)
-        offers = Offer.where("extract(year from resolved_at) = ? AND extract(month from resolved_at) = ? AND status = ?", year, month, "accepted")
-        offers.select do |offer| 
-            offer.custom_fields['employment_type'] == "Full-time" and offer.job_id != 571948
-        end 
+
+    def self.get_accepted_offers_for_year_and_month(year, month)
+        Offer.where("extract(year from resolved_at) = ? AND
+                     extract(month from resolved_at) = ? AND
+                     status = ? AND 
+                     custom_fields ->> 'employment_type' = ? AND
+                     job_id != ?", year, month, 'accepted', 'Full-time', 571948)
```

Comment:
what is 571948? it seems to be an id. for what?

Reply:
it's an id for a job that hr wants to filter out